### PR TITLE
Us4 parent feature burulai khashayar

### PR DESCRIPTION
**Pull request by khashayarshayesteh**
Please review our code. Thanks

### CODE REVIEW

File: src/main/java/com/cydeo/service/ParentService.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+public class ParentService implements CRUDService<Parent>{
+    @Override
+    public Parent findById(int id) {
+        return parentList.stream().filter(parent -> parent.getId()==id)
+                .findAny().get();
+    }
+
+    @Override
+    public List<Parent> findAll() {
+        return new ArrayList<>(parentList);
```
Comment (Cemil5) [inline src/main/java/com/cydeo/service/ParentService.java:23]: you can return just parentList without creating new ArrayList

File: src/main/java/com/cydeo/service/ParentService.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+
+
+
+
+    }
+
+    @Override
+    public void deleteById(int id) {
+        parentList.removeIf(P->P.getId()==id);
```
Comment (Cemil5) [inline src/main/java/com/cydeo/service/ParentService.java:55]: you should use lowercase variable inside removeIf

File: src/main/java/com/cydeo/service/ParentService.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+    @Override
+    public void save(Parent parent) {
+
+        parentList.add(parent);
+
+    }
+
+    @Override
+    public void update(Parent parent) {
```
Comment (Cemil5) [inline src/main/java/com/cydeo/service/ParentService.java:35]: it is better to delete blank lines.

**Review by Cemil5 — APPROVED** [review]
Thanks for your efforts.
I approved this with some comments. please accept them as a small feedback

**Review by DilaraAsl — COMMENTED** [review]
Approved. Thanks

**Review by Cemil5 — APPROVED** [review]

**Review by DilaraAsl — APPROVED** [review]
Well done!

**Comment by Zhibektoktogulova** [comment]
Great job

**Pull request merged**